packages/my-extended-retail-app/config/default.js

```diff
@@ -30,10 +30,11 @@ module.exports = {
             }
         },
         einsteinAPI: {
-            proxyPath: `/mobify/proxy/einstein`,
+            host: 'https://api.cquotient.com',
```

Comment:
should we fix this in the base template?

Reply:
The change is already in the base template @alexvuong .

Probably what happened was that the extension app was generated before the change to the base template was made.

Reply:
See https://github.com/SalesforceCommerceCloud/pwa-kit/blob/fix-einstein-in-extensible-template/packages/template-retail-react-app/config/default.js#L33